astro/src/content/docs/get-started/core-concepts/tenants.mdx

```diff
@@ -321,6 +321,15 @@ Once you have configured your email settings, you may test your configuration wi
     * Disabled. A challenge will not be required during login.
     * Required. A challenge will be required during login. &nbsp; <AvailableSince since="1.42.0"/>
   </APIField>
+  <APIField name="MFA requirement lambda" optional>
+    The MFA requirement lambda that will be invoked during logins, password changes, and MFA Status API calls. FusionAuth uses this lambda to determine whether to challenge the user with MFA.
```

Comment:
```suggestion
    The MFA requirement lambda that will be invoked during logins, password changes, and MFA Status API calls. If set, overrides the any MFA requirement lambda provided by the application. FusionAuth uses this lambda to determine whether to challenge the user with MFA.
```
I think this is true, but you should check my logic. I think it's worth clarifying because some users might assume that we run both and OR them, or that the override works the opposite way.

Reply:
Gotcha, thanks for clarifying. Then should we change it this way instead?

```suggestion
    The MFA requirement lambda that will be invoked during logins, password changes, and MFA Status API calls. FusionAuth uses this lambda to determine whether to challenge the user with MFA. Will be overridden by any MFA requirement lambda provided by the application.
```

Also changed the ordering, since I think we should explain the use before the override.